### PR TITLE
feat(nimbus): Update models for localized experiments

### PR DESCRIPTION
**Pull request by brennie**
Because

- we want to support localized experiments in Nimbus

this commit

- updates our models to support localizations.

Co-authored-by: Jason Prickett <jprickett@mozilla.com>

### CODE REVIEW

File: experimenter/experimenter/experiments/models.py [hunk 1]
```diff
@@ -209,6 +209,9 @@ class NimbusExperiment(NimbusConstants, TargetingConstants, FilterMixin, models.
     prevent_pref_conflicts = models.BooleanField(blank=True, null=True, default=False)
     proposed_release_date = models.DateField(blank=True, null=True)
 
+    is_localized = models.BooleanField(default=False)
+    localizations = models.TextField(blank=True, null=True)
```
Comment (yashikakhurana) [inline experimenter/experimenter/experiments/models.py:213]: we can use JSONField here, thoughts 🤔 ?

**Review by yashikakhurana — COMMENTED** [review]
Generally look good, just wondering why not use JSONField in place of the text field so that we can take advantage of json validation, serialization, querying etc etc

File: experimenter/experimenter/experiments/models.py (hunk 1 above)
Comment (jaredlockhart) [reply experimenter/experimenter/experiments/models.py:213]: Exactly correct @brennie but a good thought @yashikakhurana 👍   We used to use JSON field for this but we had to move away from it.

**Review by jaredlockhart — APPROVED** [review]
👍 looks good thnx @brennie 🎉

**Pull request merged**